packages/frontend/src/components/select_sports.tsx

```diff
@@ -0,0 +1,61 @@
+import { Sport } from "@esp-group-one/types";
+import { useState } from "react";
+import type { ChangeEvent } from "react";
+
+export function SelectSport() {
+  const [sport, setSport] = useState("Tennis");
+  const [colour, setColour] = useState("tennis");
+
+  const options = [
+    { colour: "tennis", label: "Tennis" },
+    { colour: "badminton", label: "Badminton" },
+    { colour: "squash", label: "Squash" },
```

Comment:
Please use the `Sport.Tennis` etc.

It would also be useful to input a given list of sports and have this generated (as some people may not be doing all sports).

You can then convert the `Sport` to string with `as string` and then uppercase the first letter to get the label